tests/lib/bootstrap.js

```diff
@@ -102,6 +107,19 @@ function jqueryUrl() {
 	return url;
 }
 
+function migrateUrl() {
+	var version = parseUrl().migrate;
+	var url;
+
+	if ( version === "git" ) {
+		url = "http://code.jquery.com/jquery-migrate-" + version;
+	} else {
+		url = "../../../external/jquery-migrate-" + ( version || "1.12.4" ) + "/jquery-migrate";
```

Comment:
Updated with your suggestion to resolve jquery-migrate version depending on jquery version.